api/src/main/java/org/openmrs/module/fhir2/api/search/param/SearchParameterMap.java

```diff
@@ -56,10 +56,6 @@ public SearchParameterMap addParameter(@Nonnull String key, @Nonnull Serializabl
 	 * @return {@link org.openmrs.module.fhir2.api.search.param.SearchParameterMap}
 	 */
 	public SearchParameterMap addParameter(@Nonnull String key, String propertyName, @Nonnull Serializable param) {
-		if (key == null || param == null) {
```

Comment:
ide may have accidentally changed this, but it does seem like it's an unnecessary if statement with the Nonnull annotations (unless those aren't guaranteed to throw an exception in some cases?)